backend-api/src/functions/common/lambdaResponses.ts

```diff
@@ -1,5 +1,19 @@
 import { APIGatewayProxyResult } from "aws-lambda";
 
+export const badRequestResponse = (
+  error: string,
+  errorDescription: string,
+): APIGatewayProxyResult => {
+  return {
```

Comment:
I think we need security headers here

Reply:
Okay I shall I add, I wasn't sure whether the security headers for the 501 were specific as listed on the ticket. Should all responses from this lambda give the same security headers?

Reply:
Added

Reply:
Added an assertion in the API test to check for them